gatorgrade/__init__.py

```diff
@@ -0,0 +1 @@
+"""GatorGrade is a program to do X."""
```

Comment:
My bad on this suggestion... I meant that you should replace `X` with what GatorGrade does at a high-level, which should be clear enough at this point I think.